src/chrome/content/rules/nexusmods.com.xml

```diff
@@ -5,52 +5,18 @@
 
 <ruleset name="NexusMods.com">
 	<target host="nexusmods.com" />
-	<target host="www.nexusmods.com" />
-	<target host="chivalrymw.nexusmods.com" />
-	<target host="darksouls.nexusmods.com" />
-	<target host="dev.nexusmods.com" />
-	<target host="devdocs.nexusmods.com" />
-	<target host="dragonage.nexusmods.com" />
-	<target host="dragonage2.nexusmods.com" />
-	<target host="dungeondefenders.nexusmods.com" />
-	<target host="fallenenchantress.nexusmods.com" />
-	<target host="fallout3.nexusmods.com" />
-	<target host="farcry.nexusmods.com" />
-	<target host="filedelivery.nexusmods.com" />
-	<target host="forum.nexusmods.com" />
-	<target host="forums.nexusmods.com" />
-	<target host="global.nexusmods.com" />
-	<target host="grimrock.nexusmods.com" />
-	<target host="mbwarband.nexusmods.com" />
-	<target host="mbwithfireandsword.nexusmods.com" />
-	<target host="morrowind.nexusmods.com" />
-	<target host="mountandblade.nexusmods.com" />
-	<target host="neverwinter.nexusmods.com" />
-	<target host="neverwinter2.nexusmods.com" />
-	<target host="newvegas.nexusmods.com" />
-	<target host="oblivion.nexusmods.com" />
-	<target host="pw.nexusmods.com" />
-	<target host="reskin.nexusmods.com" />
-	<target host="shadowrunreturns.nexusmods.com" />
-	<target host="skyrim.nexusmods.com" />
-	<target host="static-1.nexusmods.com" />
-	<target host="static-2.nexusmods.com" />
-	<target host="static-3.nexusmods.com" />
-	<target host="static-4.nexusmods.com" />
-	<target host="static-5.nexusmods.com" />
-	<target host="static-6.nexusmods.com" />
-	<target host="static-7.nexusmods.com" />
-	<target host="static-8.nexusmods.com" />
-	<target host="staticdelivery.nexusmods.com" />
-	<target host="tes.nexusmods.com" />
-	<target host="wfto.nexusmods.com" />
-	<target host="witcher.nexusmods.com" />
-	<target host="witcher2.nexusmods.com" />
-	<target host="worldoftanks.nexusmods.com" />
-	<target host="wot.nexusmods.com" />
-	<target host="x.nexusmods.com" />
-	<target host="xcom.nexusmods.com" />
-	<target host="xrebirth.nexusmods.com" />
+	<target host="*.nexusmods.com" />
+
+	<!-- Test URLs -->
+	<test url="http://nexusmods.com/" />
+	<test url="http://www.nexusmods.com/" />
```

Comment:
Since we have self tests against listed hosts, this is a duplicate test url. Please remove this line